src/components/node/stanford-page/home-page/today-hours.tsx

```diff
@@ -127,12 +127,12 @@ const TodayLibraryHours = ({branchId}: {branchId?: string}) => {
     )
   }
   const {closedAllDay, isOpen, openingTime, closingTime, afterClose} = libraryHours
-  const hoursDisplay = !closedAllDay && (isOpen ? "Closes at " + closingTime : afterClose ? "Closed at " + closingTime : "Opens at " + openingTime)
+  const hoursDisplay = !closedAllDay && (isOpen ? "Open until " + closingTime : afterClose ? "Closed until " + openingTime : "Closed until " + openingTime)
```

Comment:
opening time is not the time tomorrow. It is the opening time of the current day. This will lead to incorrect information.

Reply:
Oh I see because the days could have different opening times.  They don't necessarily open at 8am every day, for example.

Reply:
Yes. And some times the next day isn't even open, or we don't have the information for. Such as on saturdays, we don't know what time the location opens on Sunday, if at all